nltk/classify/rte_classify.py

```diff
@@ -32,7 +32,7 @@ def ne(token):
         return True
     return False
 
-def lemmatize(word):
+def _lemmatize(word):
```

Comment:
@jhelie: would you please comment on this change? How do we know `lemmatize` isn't being used externally?